Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -680,21 +708,54 @@ CGPathRef CGPathCreateWithRoundedRect(CGRect rect, CGFloat cornerWidth, CGFloat
 }
 
 /**
- @Status Stub
- @Notes
+ @Status Interoperable
 */
 bool CGPathEqualToPath(CGPathRef path1, CGPathRef path2) {
-    UNIMPLEMENTED();
-    return StubReturn();
+    if (path1->_count != path2->_count) {
+        return false;
+    }
+
+    for (unsigned i = 0; i < path1->_count; i++) {
+        CGPathElement* element1 = &path1->_elements[i];
+        CGPathElement* element2 = &path2->_elements[i];
+        if (element1->type != element2->type) {
+            return false;
+        }
+        unsigned pointCount = _CGPathPointCountForElementType(element1->type);
+        for (unsigned p = 0; p < pointCount; p++) {
+            CGPoint p1 = element1->points[p];
```

Comment:
You can avoid creating copies of p1 and p2 by expanding the expressions below or using ref/pointer instead.